cosmo_tester/test_suites/manager_of_managers/regional_cluster.py

```diff
@@ -15,12 +15,14 @@
 
 import os
 import json
+from contextlib import contextmanager
 
 from cloudify_cli.constants import DEFAULT_TENANT_NAME
+from fabric import api as fabric_api
```

Comment:
Please order by length and separate 3rd package from our own

Reply:
sure